### PR TITLE
create a p12 certificate for local dev with ssl

### PR DESCRIPTION
**Pull request by jygaulier**
create a p12 certificate for local dev with ssl
ask for passphrase only once
[SKIP CI]

### CODE REVIEW

File: infra/ssl/create-root-ca.sh [hunk 1]
```diff
@@ -2,9 +2,19 @@
 set -e
 
 mkdir -p ~/ssl/
-openssl genrsa -des3 -out ~/ssl/AlchemyRootCA.key 2048
+read -s -p "Enter passphrase: " pass
```
Comment (4rthem) [inline infra/ssl/create-root-ca.sh:5]: Good idea!

File: doc/configure-ssl.md [hunk 2]
```diff
@@ -13,4 +13,4 @@ sudo service nginx configtest
 sudo service nginx reload
 ```
 
-Then you can add the AlchemyRootCA (located at `~/ssl/AlchemyRootCA.pem`) in your favorite browser. 
+Then you can add the AlchemyRootCA (located at `~/ssl/AlchemyRootCA.p12`) in your favorite browser.
```
Comment (4rthem) [inline doc/configure-ssl.md:16]: `.pem` is working for me in Chrome, why do you need it?

Comment (jygaulier) [reply doc/configure-ssl.md:16]: Version 88.0.4324.150 (Official Build) (64-bit)

pem don't show up if i don't select "all files", and error when imported.
image
image
image

p12 is fine.

Comment (4rthem) [reply doc/configure-ssl.md:16]: You need to import the certificate as an Authority one:
image

Comment (jygaulier) [reply doc/configure-ssl.md:16]: ok, it works.
so we can remove  the p12 generation, but add a few words in the doc ("install pem for dummies" ?)

**Pull request merged**